database/dataGenerator.js

```diff
@@ -0,0 +1,65 @@
+const faker = require('faker');
+
+// generate n fake hotels
+const generate = (n) => {
+  var hotels = [];
+  for (var i = 0; i < n; i++) {
+    var currHotelReview = {
+      responderInfo: {},
+      memberInfo: {},
+      reviewInfo: {}
+    };
+
+    let travelTypes = ['Families','Couples', 'Solo', 'Business', 'Friends'];
+    let close = faker.company.bsAdjective();
+    //hotel info
+    currHotelReview.responderInfo.hotelId = i;
+    currHotelReview.responderInfo.responderOrg = faker.company.companyName();
+    currHotelReview.responderInfo.responderPicture = `https://adcobareviews.s3-us-west-1.amazonaws.com/a30.jpg`;
+    currHotelReview.responderInfo.responderClose = close.charAt(0).toUpperCase() + close.slice(1);
+
+    var randomReviewNumber = Math.ceil(Math.random() * 5);
+    //change back
+    for (var j = 0; j < 5; j++) {
+      const randomDate = faker.date.recent(360);
+      currHotelReview.responderInfo.responderDate = randomDate;
+      currHotelReview.responderInfo.responderName = faker.name.findName();
+      currHotelReview.responderInfo.responderPosition = faker.name.jobTitle();
+      currHotelReview.responderInfo.responderText = faker.lorem.paragraph(3, '\n');
+      //member info
+      currHotelReview.memberInfo.memberId = j;
+      currHotelReview.memberInfo.memberImg = `https://adcobareviews.s3-us-west-1.amazonaws.com/a${Math.ceil((Math.random() * 9) + 20)}.jpg`;
+      currHotelReview.memberInfo.memberUserName = faker.internet.userName();
+      currHotelReview.memberInfo.memberLocation = faker.address.city();
+      currHotelReview.memberInfo.memberContributions = Math.ceil(Math.random() * 50);
+      currHotelReview.memberInfo.memberHelpful = Math.floor(Math.random() * 5);
```

Comment:
### Quick note on readability
This is definitely just a preference thing, so there's no need to change it if you don't want to, but as an alternative you could declare these variables as follows:
```
currHotelReview.memberInfo = {
  memberId: j,
  memberImg: `https://adcobareviews.s3-us-west-1.amazonaws.com/a${Math.ceil((Math.random() * 9) + 20)}.jpg`
  memberUserName: faker.internet.userName(),
  memberContributions: Math.ceil(Math.random() * 50),
  memberHelpful: Math.floor(Math.random() * 5)
}
```
### If you wanted to take it even further...
You could declare all of the variables and then use destructuring to streamline it even more.
```
memberId = j;
memberImg = `https://adcobareviews.s3-us-west-1.amazonaws.com/a${Math.ceil((Math.random() * 9) + 20)}.jpg`;
memberUserName = faker.internet.userName();
memberLocation = faker.address.city();
memberContributions = Math.ceil(Math.random() * 50);
memberHelpful = Math.floor(Math.random() * 5);

currHotelReview.memberInfo = { memberId, memberImg, memberUserName, memberLocation, memberContributions, memberHelpful };
```

#### You could apply this to `reviewInfo` and `responderInfo` as well